feature-wallet-impl/src/main/java/jp/co/soramitsu/wallet/impl/presentation/cross_chain/setup/CrossChainSetupViewModel.kt

```diff
@@ -104,6 +103,7 @@ class CrossChainSetupViewModel @Inject constructor(
     val openValidationWarningEvent: LiveData<Event<Pair<TransferValidationResult, ValidationWarning>>> = _openValidationWarningEvent
 
     private val payload: AssetPayload? = savedStateHandle[CrossChainSetupFragment.KEY_PAYLOAD]
+    private val walletIdFlow = MutableStateFlow<Long?>(null)
```

Comment:
What is this walletIdFlow is used for? If we store here current selected wallet, let's better call it `selectedWalletIdFlow`